docs/reference/mapping/dynamic/templates.asciidoc

```diff
@@ -124,18 +128,55 @@ PUT my-index-000001/_doc/1
 <1> The `my_integer` field is mapped as an `integer`.
 <2> The `my_string` field is mapped as a `text`, with a `keyword` <<multi-fields,multi field>>.
 
+[[match-mapping-runtime-fields]]
+===== Mapping runtime fields in a dynamic template
+You can also map runtime fields in a dynamic template. For example, the
+following request adds a dynamic template named `dayOfWeek` that maps all
+incoming `string` fields as `keyword` runtime fields. Although the `runtime`
+definition is blank, new `string` fields will be mapped as `keyword` runtime
+fields based on the <<dynamic-field-mapping-types,dynamic mapping rules>> that
+{es} uses for adding field types to the mapping. Any `string` that doesn't pass
+date detection or numeric detection is automatically mapped as a `keyword` when
```

Comment:
as `keyword` within the runtime section?

Reply:
Based on the [dynamic mapping rules](https://elasticsearch_66530.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/dynamic-field-mapping.html#dynamic-field-mapping), a string that doesn’t pass date detection or numeric detection is mapped as a `keyword` for runtime fields, correct?

Reply:
yes, I was referring to the fact that keyword (as well as other field types) exist both under runtime and properties. So I was wondering if we need to be explicit about which side of things we are talking about

Reply:
We can be more explicit and say:

>For example, the following request adds a dynamic template named `dayOfWeek` that maps all
incoming `string` fields as `keyword` fields in the `runtime` section of the mapping.

Reply:
++